extensions/pkg/webhook/certificates/reconciler.go

```diff
@@ -86,7 +86,12 @@ type reconciler struct {
 // AddToManager generates webhook CA and server cert if it doesn't exist on the cluster yet. Then it adds reconciler to
 // the given manager in order to periodically regenerate the webhook secrets.
 func (r *reconciler) AddToManager(ctx context.Context, mgr manager.Manager) error {
-	r.serverPort = mgr.GetWebhookServer().(*webhook.DefaultServer).Options.Port
+	defaultServer, ok := mgr.GetWebhookServer().(*webhook.DefaultServer)
+	if !ok {
+		return fmt.Errorf("expected *webhook.DefaultServer, got %T", defaultServer)
```

Comment:
For printing the type, you should use the original object, see 
https://github.com/gardener/gardener/blob/5abed1bfedd6f2188e5dcec9af161d1b5c286236/pkg/resourcemanager/webhook/seccompprofile/handler.go#L36-L39